lib/std/os/linux/mips.zig

```diff
@@ -18,6 +18,7 @@ pub fn syscall0(number: SYS) usize {
 }
 
 pub fn syscall_pipe(fd: *[2]i32) usize {
+    _ = fd;
```

Comment:
This looks like a bug

Reply:
@LemonBoy what's going on here, can we get some comments on why syscall1 doesn't work for pipe here?

Reply:
The ABI for the pipe syscall differs from every other syscall: the two FDS are returned in two different registers instead of taking a pointer to a two-element array.

Reply:
So this is missing  `          [arg1] "{$4}" (arg1)` then, yeah?